eos-updater-flatpak-installer/eos-updater-flatpak-installer.service.in

```diff
@@ -1,7 +1,7 @@
 [Unit]
 Description=Endless OS Post-Boot Flatpak Installer
-Wants=ostree-remount.service
-After=ostree-remount.service
+Wants=ostree-remount.service eos-extra-settled.target
```

Comment:
@ramcq Sorry, just thought about this again. Should this be `Requires: eos-extra-settled.target` to ensure that the SD card is actually mounted?

Reply:
I stared at the manpage and couldn't figure out what I really needed here. I flipped back and forth between Wants and Requires. I also got distracted by the unit not just depending on local-fs.target. :)